docs/developers/customize-karmada-scheduler.md

```diff
@@ -187,4 +187,167 @@ kubectl kubectl --kubeconfig ~/.kube/karmada.config --context karmada-host edit
         - --v=4
         image: ## Your Image Address
 ...
+```
+
+## Configure Multiple Schedulers
+
+### Run the second scheduler in the cluster
+
+You can run multiple schedulers simultaneously alongside the default scheduler and instruct Karmada what scheduler to use for each of your workloads.
+Here is an sample of the deployment config. You can save it as `my-scheduler.yaml`:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-karmada-scheduler
+  namespace: karmada-system
+  labels:
+    app: my-karmada-scheduler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: my-karmada-scheduler
+  template:
+    metadata:
+      labels:
+        app: my-karmada-scheduler
+    spec:
+      automountServiceAccountToken: false
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+      containers:
+        - name: karmada-scheduler
+          image: docker.io/karmada/karmada-scheduler:latest
+          imagePullPolicy: IfNotPresent
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 10351
+              scheme: HTTP
+            failureThreshold: 3
+            initialDelaySeconds: 15
+            periodSeconds: 15
+            timeoutSeconds: 5
+          command:
+            - /bin/karmada-scheduler
```

Comment:
Does the customized scheduler also named `karmada-scheduler`?

Reply:
This is the name of the compiled binary. Changing to `karmada-scheduler` will not affect the custom scheduler logic. The same as https://kubernetes.io/zh-cn/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#%E4%B8%BA%E8%B0%83%E5%BA%A6%E5%99%A8%E5%AE%9A%E4%B9%89-kubernetes-deployment.